### PR TITLE
Add qiankun technology

### PR DESCRIPTION
**Pull request by kuitos**
## website
https://qiankun.umijs.org/

## github repo
https://github.com/umijs/qiankun

## examples
Sorry about the missing of examples, as most of the time this technology is used in enterprise, who are unaccessible from outside the company.

### CODE REVIEW

**Comment by kuitos** [comment]
Already tested it locally, wish to be merged😄
image

File: src/technologies/q.json [hunk 1]
```diff
@@ -315,5 +315,21 @@
...
+      59
+    ],
+    "description": "qiankun is a JS library who helps developers to build a micro frontends system.",
+    "dom": {
+      "div[id^=__qiankun_]": {
+        "attributes": {
+          "data-version": "([\\d\\.]+)\\;version:\\1"
+        }
+      }
+    },
```
Comment (nurbek91) [inline src/technologies/q.json:330]: we can add also js 

```
"js": {
      "qiankun.loadMicroApp": ""
    },
```

example https://www.medsci.cn/

File: src/technologies/q.json [hunk 2]
```diff
@@ -315,5 +315,21 @@
...
+    "icon": "qiankun.svg",
```
Comment (nurbek91) [inline src/technologies/q.json:331]: [qiankun.svg.zip](https://github.com/wappalyzer/wappalyzer/files/9168631/qiankun.svg.zip)
32x32

File: src/technologies/q.json [hunk 3]
```diff
@@ -315,5 +315,21 @@
...
     "oss": true,
     "website": "https://qwik.builder.io"
+  },
+  "qiankun": {
+    "cats": [
+      59
+    ],
+    "description": "qiankun is a JS library who helps developers to build a micro frontends system.",
+    "dom": {
+      "div[id^=__qiankun_]": {
```
Comment (nurbek91) [inline src/technologies/q.json:325]: `"div[id^=__qiankun_]":`

`"div[id^='__qiankun_']":`

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!
Small updates required

File: src/technologies/q.json (hunk 3 above)
Comment (kuitos) [reply src/technologies/q.json:325]: Are these two not equivalent?

File: src/technologies/q.json (hunk 2 above)
Comment (kuitos) [reply src/technologies/q.json:331]: done

File: src/technologies/q.json (hunk 1 above)
Comment (kuitos) [reply src/technologies/q.json:330]: That might not be accurate, normally we would not attach qiankun to a global variable.
But the dom selector must be accurate.

File: src/technologies/q.json (hunk 3 above)
Comment (kuitos) [reply src/technologies/q.json]: Added the single quotes to keep consistency with others, thanks for your quick response~

File: src/technologies/q.json [hunk 4]
```diff
@@ -315,5 +315,22 @@
...
+    ],
+    "description": "qiankun is a JS library who helps developers to build a micro frontends system.",
+    "dom": {
+      "div[id^='__qiankun_']": {
+        "attributes": {
+          "data-version": "([\\d\\.]+)\\;version:\\1"
+        }
+      }
+    },
+    "scripts": "__POWERED_BY_QIANKUN__",
```
Comment (kuitos) [inline src/technologies/q.json:331]: Added a source code detection rule, could you pls donate some times to review it? @nurbek91

Comment (nurbek91) [reply src/technologies/q.json:331]: Hey @kuitos ,
This option is less resource-intensive parameter
```
"js": {
      "qiankun.loadMicroApp": ""
    },
```

**dom** and **scripts** the most resource-intensive parameters.
From the technical side, everything looks correct.

But let's leave the decision @AliasIO , this is where my powers end.

Comment (kuitos) [reply src/technologies/q.json:331]: > This option is less resource-intensive parameter

Yes it is, but it‘s not common usage, so it's not very accurate.

@AliasIO Could you pls donate some time to review this pr?

**Comment by AliasIO** [comment]
Thanks!

**Comment by kuitos** [comment]
Not to rush you, just wondering when we could release a new version for chrome extension? @AliasIO 😀

**Comment by AliasIO** [comment]
Today :)

**Pull request merged**